_episodes_rmd/01-raster-structure.Rmd

```diff
@@ -447,9 +447,7 @@ ggplot() +
 
 Notice that a warning message is thrown when R creates the histogram.
 
-`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
-
-This warning is caused by `ggplot`'s default settings enforcing that there are
+This warning is caused by `ggplot2`'s default settings enforcing that there are
```

Comment:
Should this be `geom_histogram` not `ggplot` or `ggplot2`?

Reply:
You're right, I thought of that too. I think it would be equally right, but more specific to use `geom_histogram()`. Either way would be good, as long as it's not `ggplot`! :)